sdk/finding.ts

```diff
@@ -78,6 +91,9 @@ export class Finding {
     // TODO assert metadata keys and values are strings
 
     labels = labels.map(l => l instanceof Label ? l : Label.fromObject(l))
-    return new Finding(name, description, alertId, protocol, severity, type, metadata, addresses, labels)
+    const sources = {chainSource: {chainId: chainId}}
```

Comment:
i user does not specify chainId input, should this be undefined as well?

Reply:
yes